src/interface/storage.thrift

```diff
@@ -507,6 +507,8 @@ struct IndexColumnHint {
     2: ScanType                 scan_type,
     3: common.Value             begin_value,
     4: common.Value             end_value,
+    5: bool                     include_begin = true,
+    6: bool                     include_end = false,
```

Comment:
Please give some comments . 
And I think instead of this to optional  3 , 4 is better.

Reply:
Good idea.I will add some notes later. But modifying the serial number of the members in IDL is not a good suggestion, it may cause unexpected errors.